src/steps/user/index.test.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  createMockStepExecutionContext,
+  Recording,
+} from '@jupiterone/integration-sdk-testing';
+
+import { setupSigSciRecording } from '../../../test/recording';
+
+import { IntegrationConfig } from '../../config';
+import { fetchUsers } from './';
+import { integrationConfig } from '../../../test/config';
+import { createOrganizationEntity } from '../organization/converter';
+import { SignalSciencesCorp } from '../../types';
+
+describe('should collect user data', () => {
+  let recording: Recording;
+
+  beforeEach(() => {
+    recording = setupSigSciRecording({
+      directory: __dirname,
+      name: 'buildCorpAndUserRelationship',
+    });
+  });
+
+  afterEach(async () => {
+    await recording.stop();
+  });
+
+  test('validate client integration and converters', async () => {
+    // Arrange
+    const context = createMockStepExecutionContext<IntegrationConfig>({
+      instanceConfig: integrationConfig,
+    });
+    await context.jobState.addEntity(
+      createOrganizationEntity({
+        name: 'jupiterone',
+        displayName: 'JupiterOne',
+      } as SignalSciencesCorp),
+    );
+
+    // Act
+    await fetchUsers(context);
+
+    // Assert
+    expect({
+      numCollectedEntities: context.jobState.collectedEntities.length,
+      numCollectedRelationships: context.jobState.collectedRelationships.length,
+      collectedEntities: context.jobState.collectedEntities,
+      collectedRelationships: context.jobState.collectedRelationships,
+      encounteredTypes: context.jobState.encounteredTypes,
+    }).toMatchSnapshot();
```

Comment:
tip: You can test relationships using the custom `toMatchDirectRelationshipSchema` matcher (similar to how we test entities):

```ts
const corpHasUserRelationships = context.jobState.collectedRelationships;
expect(corpHasUserRelationships.length).toEqual(users.length);
expect(corpHasUserRelationships).toMatchDirectRelationshipSchema({
  schema: { 
    properties: { 
      _type: { 
        const: Relationships.ORGANIZATION_HAS_USER._class 
      } 
    } 
  }
});
```